cmd/nerdctl/builder_build_test.go

```diff
@@ -279,8 +279,8 @@ CMD echo $TEST_STRING
 		})
 	}
 
-	t.Run("InvalidBuildArgCausesError", func(t *testing.T) {
-		base.Cmd("build", buildCtx, "-t", imageName, "--build-arg", "=TEST_STRING").AssertFail()
+	t.Run("InvalidBuildArgNotCausesError", func(t *testing.T) {
+		base.Cmd("build", buildCtx, "-t", imageName, "--build-arg", "=TEST_STRING").AssertOK()
```

Comment:
Seems to be a bug of Docker

Reply:
Set Buildx and BuildKit as the default builder on Linux. [moby/moby#43992](https://github.com/moby/moby/pull/43992) from https://docs.docker.com/engine/release-notes/23.0/.

So there are some change of the build-args.

Should the test be removed , because it may be a bug :-)

Reply:
Reported:
- https://github.com/docker/cli/issues/4494

The test can be removed